less/index.less

```diff
@@ -54,6 +54,103 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size: 62.5%;
+	
 }
 
+.container {
+	max-width: 880px;
+	font-size:1.6rem;
+	margin: 0 auto;
+	font-family: @fontStack;
+}
+
+@mainColor: darkgrey;
+@fontStack: Helvetica, Arial;
+
```

Comment:
hi gannon, you may want to setup the less file structure with the variables on top just for easier readability

Reply:
Will do. 

and it's my buddy from college lol.